examples/src/main.rs

```diff
@@ -77,7 +77,15 @@ fn main() {
     let proof_bytes = proof.to_bytes();
     debug!("Proof size: {:.1} KB", proof_bytes.len() as f64 / 1024f64);
     let security_level = options.get_proof_security_level(&proof);
-    debug!("Proof security: {} bits", security_level);
+    debug!(
+        "Proof security: {} bits ({})",
+        security_level,
+        if options.proven_security {
+            "proven security"
+        } else {
+            "conjectured security"
+        }
+    );
```

Comment:
One other thought I had: what if instead of selecting proof security type via CLI argument, we just print out both levels. Maybe the output line could look something like this:
```
Proof security: 96 bits (40 bits proven)
```

Reply:
I had actually went for it originally, but thought we wouldn't want to display the proven security by default as it may look quite scary. But if we think it's not a problem, then I can revert it!

Reply:
Yeah - I thought about it too. But I guess it is better to show it and then people can decide for themselves which security assumptions they are comfortable with.

Reply:
I'm calling the function twice (for both conjectured and proven), would you rather have the utility method returning some tuple (conjectured, proven) directly instead?

Reply:
I think it is fine to call it twice as you have now.